Resources/ServerInfo/Guidebook/DeltaV/AlertProcedure.xml

```diff
@@ -0,0 +1,90 @@
+<Document>
+# Alert Procedure
+Alert levels exist on the station to advise the crew at-large of the state of the station's overall security or well-being. Setting an alert level via the communications computer is an immediate action, so it is often done after sending a message to the crew.
+<Box>
+<GuideEntityEmbed Entity="ComputerComms" Caption="This sets the alert level."/>
+<GuideEntityEmbed Entity="EmergencyLight" Caption="This displays the alert level."/>
+</Box>
+There are some restrictions on when certain alert levels can be used. They should be somewhat intuitive, but they are elaborated on as follows:
+
+## Code Green
+[color=#7cfc00]Regular operating conditions, absent or manageable station threat(s) present.[/color]
+- [color=#a4885c]Chain of Command (COC)[/color]: Captain ◃ Department Heads
+- [color=#a4885c]Armory Policy[/color]: Open carrying of lethal arms disallowed. Standard ballistic armor recommended for all security personnel.
+- [color=#a4885c]Discipline[/color]: Report to supervisor for general orders.
+- [color=#a4885c]Secure Areas[/color]: Secure areas unlocked and accessible to all authorized personnel. High security areas and areas with an expectation of privacy, such as bedrooms, dorms, or medical treatment facilities, may be locked as needed.
+- [color=#a4885c]Medical[/color]: Standard practice.
+- [color=#a4885c]Engineering[/color]: Standard practice.
+- [color=#a4885c]Rules of Engagement (ROE)[/color]: Engage with minimal force; prioritize de-escalation whenever possible.
+
+## Code Blue
+[color=#0000FF]Elevated alert status. There is a confirmed threat to the station or its crew.[/color]
+- [color=#a4885c]COC[/color]: Captain ◃ Head of Security ◃ Department Heads
+- [color=#a4885c]Armory Policy[/color]: Open carrying lethal weaponry permitted for all security personnel. Standard ballistic armor mandatory for security personnel, soft armor authorized for all other personnel.
```

Comment:
I'm not sure what soft armour is, are they EVA suits?
Also might want to add that open carry of lethals isn't necessarily recommended, even though it is permitted? idk